esp/src/eclwatch/WsWorkunits.js

```diff
@@ -141,11 +141,6 @@ define([
             return ESPRequest.send("WsWorkunits", "WUGetNumFileToCopy", params);
         },
 
-        CreateQuerySetStore: function (options) {
```

Comment:
What happens to anything calling this? Or does the linting look for calls that are unused?